manifests/entry.pp

```diff
@@ -2,8 +2,8 @@
   $type,
   $options,
   $ensure = $backupninja::ensure,
```

Comment:
What type are these parameters?

From the code I can deduce that `type` must be a String but I'm not so sure for the rest of these params.

Reply:
I bet both type and ensure are Enum (see other modules) and options maybe Hash

Reply:
You are right, fix implemented an now there are no typeless parameters.